test/unit/attachment.js

```diff
@@ -44,7 +44,9 @@ describe('Attachment', function() {
       newAttachment.fileName.should.be.equal(file.name)
 
       newAttachment.should.have.a.property('fileSize')
-      newAttachment.fileSize.should.be.equal(file.size.toString())
+      if (file.size >= 0) {
+        newAttachment.fileSize.should.be.equal(file.size.toString())
+      }
```

Comment:
I understand we need this because we don't know the new size for re-rotated image (and it can be anything with various gm versions). But could we be, like, less "loose", and at least check for the new size being greater than zero or something? WDYT?